src/pfe/portal/modules/utils/kubernetesFunctions.js

```diff
@@ -151,4 +200,7 @@ module.exports = {
   patchProjectDeployments,
   getProjectIngress,
   getServicePortFromProjectIngress,
+  getProjectRoute,
+  getServicePortFromProjectRoute,
+  getServicePortFromProjectIngressOrRoute,
```

Comment:
Are the last two new functions are only exported so the tests can access them?

Reply:
Not on purpose but yes the tests are the only thing that calls them. I'll change this.